src/components/TooltipNew/SpotlightOverlay.js

```diff
@@ -81,7 +99,8 @@ SpotlightOverlay.propTypes = {
     height: PropTypes.number.isRequired
   }).isRequired,
   isVisible: PropTypes.bool.isRequired,
-  slide: PropTypes.bool.isRequired
+  slide: PropTypes.bool.isRequired,
+  disableClicks: PropTypes.bool.isRequired
```

Comment:
There is a lot of required props, in my opinion we should provide default value here or in higher component. Btw, `disableClicks` is required here, but it isn't required at `UserGuideTooltop`.

Reply:
ooops, you are right. No need for required and default props for `disableClicks` property

Reply:
ok